pkg/ui/src/components/pagination/pagination.tsx

```diff
@@ -63,3 +65,18 @@ export function PaginationComponent(props: PaginationComponentProps) {
     />
   );
 }
+
+// tslint:disable-next-line: no-shadowed-variable
+export const paginationPageCount = (pagination: PaginationSettings, pageName: string, match?: match<any>, appAttr?: string, search?: string) => {
```

Comment:
@elkmaster this looks good but can you add some unit tests for this function to demonstrate what it does when page is 0,1, etc. and we have 0,1,2,200 results.